README.md

```diff
@@ -192,6 +192,7 @@ Options:
 - `descriptions <path>` - Required
 - `regex [name]` - Optional
 - `logFile <path>` - Optional
+- `expire <value>` - Optional
```

Comment:
compress/expire should not be added as an option here. instead it should be possible to configure it in the description yaml (this file describes the handling of the AWS S3 resources!). Also please add proper info in the README, since `expire` and `compress` doesn't give any info.